### PR TITLE
Changed website style

### PR DESCRIPTION
**Pull request by JelleTUE**
Also added some new images and replaced the membership form with the newer version

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -1,9 +1,14 @@
 /* this is atan($skew) * $site-width */
+
+/*  -webkit-transform: skewY(-10deg);
```
Comment (Melsius) [inline css/main.css]: If it's not in use anymore, just remove it. We'll always be able to get it back from the history if necessary.

Comment (Melsius) [reply css/main.css]: Applies to all other lines where these lines are commented out

File: index.html [hunk 2]
```diff
@@ -40,11 +40,12 @@
...
 <div id="main" ng-cloak>
     <a href="#">
-        <img src="images/css/logo.png" id="logo" width="365" height="250">
+        <img src="images/css/logo2.png" id="logo" width="365" height="250">
     </a>
+	
 
     <div id="header">
-        <img src="images/css/header.jpg" width="960"/>
+        <!-- <img src="images/css/header.jpg" width="960"/> -->
```
Comment (Melsius) [inline index.html]: Remove line if not necessary anymore

File: js/app.js [hunk 3]
```diff
@@ -51,6 +51,28 @@ var altTexts = {
                            eng:   'Sparring match'},
   'impressie-13.jpg':     {dutch: 'Vliegende Ilyeons',
                            eng:   'Flying Ilyeons'},
+  'impressie-14.jpg':     {dutch: 'Sfeerimpressie',
```
Comment (Melsius) [inline js/app.js]: I think you forgot to add these new files to the PR?

File: pages/home.html [hunk 4]
```diff
@@ -10,11 +10,13 @@ <h1>Home</h1>
 <p>
 	De trainingstijden zijn 
 	<ul>
-	  <li>maandag van 18:30 tot 20:00 in Hal 1</li>
-	  <li>donderdag van 20:00 tot 21:30 in Hal 3a</li>
+	  <li>Maandag van 18:30 tot 20:00 in Hal 1</li>
```
Comment (Melsius) [inline pages/home.html]: Actually, this is a continuation of the sentence "De trainingstijden zijn ...", so I don't agree that these are capitalised.

File: index.html [hunk 5]
```diff
@@ -40,11 +40,12 @@
 
 <div id="main" ng-cloak>
     <a href="#">
-        <img src="images/css/logo.png" id="logo" width="365" height="250">
```
Comment (Sc0dran) [inline index.html]: If logo.png is no longer in use, logo2.png might as well be called logo.png

File: index.html [hunk 6]
```diff
@@ -90,8 +91,9 @@
 <script src="js/app.js"></script>
 
 <!-- Script to have pak.jpg as background, requires JQuery to be imported -->
```
Comment (Sc0dran) [inline index.html]: Update comments along code, even if the impact is small

File: css/main.css [hunk 7]
```diff
@@ -40,15 +35,7 @@ td{
   padding: 0;
   margin: 0;
   height: 120px;
-/*  -webkit-transform: skewY(-10deg);
-  -moz-transform: skewY(-10deg); */
-  overflow: hidden;
-  /* the image is higher than the header itself */ }
-  #header img {
```
Comment (Sc0dran) [inline css/main.css]: Was this part intentionally deleted?

Comment (JelleTUE) [reply css/main.css]: Yeah, img part of header is no longer required

File: index.html [hunk 8]
```diff
@@ -89,9 +89,10 @@
 <script src="js/data/boards.js"></script>
 <script src="js/app.js"></script>
 
-<!-- Script to have pak.jpg as background, requires JQuery to be imported -->
+<!-- Script to have header.jpg as background, requires JQuery to be imported -->
+
 <script type="text/javascript">
-    $.backstretch("images/css/pak.jpg");
+    $.backstretch("images/css/header.jpg");
```
Comment (Melsius) [inline index.html]: Slightly confusing that the "header.jpg" is actually the whole background :p

**Comment by Melsius** [comment]
Merged already? I didn't even approve yet >.<.
Anyway, thanks for your contribution @JelleTUE!

**Pull request merged**